src/nunjucks/moduk/components/input/__examples__/with-fixed-width.njk

```diff
@@ -0,0 +1,55 @@
+{% from "moduk/components/input/macro.njk" import modukInput %}
+
+{{ modukInput({
```

Comment:
Think we'll need to wrap these in a `<div>` because the visual regression test only takes a screenshot of the first element (or maybe it should do something else when there's more than one element).